frontend/src/components/TerminalShortcuts.vue

```diff
@@ -0,0 +1,152 @@
+<!--
+Copyright (c) 2020 by SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <div>
+    <template v-if="projectShortcuts.length || shortcuts.length">
+      <template v-for="shortcut in projectShortcuts">
```

Comment:
Why not v-for on `terminal-shortcut`

Reply:
just checked, the error message is then:
```
[vue/no-use-v-if-with-v-for]
This 'v-if' should be moved to the wrapper element.
```